flavors/squid_nlb_central/updatewhitelist.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+cd /home/ubuntu/cloud-automation
+git pull
+
+DIFF_AUTH1=$(diff "/home/ubuntu/cloud-automation/flavors/squid_nlb_central/authorized_keys_admin" "/home/ubuntu/.ssh/authorized_keys")
+DIFF_AUTH2=$(diff "/home/sftpuser/cloud-automation/flavors/squid_nlb_central/authorized_keys_user" "/home/sftpuser/.ssh/authorized_keys")
+
+DIFF_SQUID1=$(diff "/home/ubuntu/cloud-automation/flavors/squid_nlb_central/web_whitelist" "/etc/squid/web_whitelist")
+DIFF_SQUID2=$(diff "/home/ubuntu/cloud-automation/flavors/squid_nlb_central/web_wildcard_whitelist" "/etc/squid/web_wildcard_whitelist")
+DIFF_SQUID3=$(diff "/home/ubuntu/cloud-automation/flavors/squid_nlb_central/ftp_whitelist" "/etc/squid/ftp_whitelist")
```

Comment:
It will be better to just check the exit code from `diff` to see if something has changed - ex:
```
echo "Checking if web_whitelist has changed"
diff ...
DIFF_SQUID1=$?
if [[ $DIFF_SQUID1 -ne 0 ]]; then ... the file has changed 
```

Reply:
done.